sdk/identity/azure-identity/CHANGELOG.md

```diff
@@ -1,14 +1,6 @@
 # Release History
 
-## 1.7.0-beta.3 (Unreleased)
-
-### Features Added
-
-### Breaking Changes
-
-### Bugs Fixed
-
-### Other Changes
+## 1.7.0-beta.3 (2024-03-07)
```

Comment:
Thanks! 3/7 and not 3/5 is intentional, correct?

Reply:
For now, yes. Giving a couple days buffer. I will check back on 3/5 and adjust if needed.